application_sandbox/conditional_rendering/main.cpp

```diff
@@ -450,16 +450,21 @@ class ConditionalRenderingSample
             Mat44::RotationY(3.14f * time_since_last_render * 0.5f));
 
     conditional_data_->data().condition =
-        (frames_since_last_notify_ % 120) < 60;
-    dispatch_data_->data().value += 1.0;
+        (frames_since_last_notify_ % 120) < 60 ? 1u : 0u;
```

Comment:
Couldn't we have just flipped the <?

Reply:
Wait nevermind, its just effectively an explicit cast

Reply:
ah, yes, sorry, it wasn't a particularly important change. It was added during debugging, I removed it now.